README.md

```diff
@@ -19,25 +19,27 @@ That's it, project has been created.  Though it would probably be good to set yo
 
 ### The app ###
 
-Now let's pull in the app source code from [GitHub repo](https://github.com/openshift/nodejs-ex) (fork if you like)
+Now let's pull in the app source code from [GitHub repo](https://github.com/openshift/nodejs-ex) (fork if you like).
 
 #### create ####
 
         $ oc new-app https://github.com/openshift/nodejs-ex -l name=myapp
         
-That should be it, `new-app` will take care of creating the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
+That should be it, this form of `new-app` will take care of pulling down the necessary image streams, and then creating the right build configuration, deployment configuration and service definition.  Next you'll be able to kick off the build, though new-app will kick off a build once all required dependencies are confirmed.  The -l flag will apply a label of "name=myapp" to all the resources created by new-app, for easy management later.
```

Comment:
it's not pulling down imagestreams, it's locating an appropriate builder image on dockerhub and creating an imagestream for that image.